manifests/install.pp

```diff
@@ -143,6 +150,11 @@
             name   => $grafana::package_name,
           }
         }
+        'FreeBSD': {
```

Comment:
why is this package required? It doesn't sound like anything grafana specific. Is the FreeBSD package for grafana lacking a dependency? If this is a generic package, it might be already managed by other modules which results in a duplicate resource error.

Reply:
Yes, the dependency already exists in the package. I just duplicated the previous entries (other distributions have no valid dependencies in grafana? I'll fix it now